prov/util/src/util_av.c

```diff
@@ -550,10 +550,16 @@ static int util_av_init(struct util_av *av, const struct fi_av_attr *attr,
 			const struct util_av_attr *util_attr)
 {
 	int *entry, i, ret = 0;
+	size_t max_count, param;
+
+	if (!fi_param_get_size_t(NULL, "universe_size", &param))
+		max_count = MAX(param, attr->count);
+	else
+		max_count = attr->count;
```

Comment:
My preference is to use attr->count when set, and only override the default value if not specified.

Reply:
Does this check work in your case instead?
```
if (attr->count) {
    max_count = attr->count;
} else {
    if (!fi_param_get_size_t(NULL, "av_size", &max_count))
        max_count = UTIL_DEFAULT_AV_SIZE;
}
```

Reply:
Yes, this works fine for me. I update the PR according your comment.
Btw, since the variable can be used in several cases (not only for AV), I named this variable `FI_UNIVERSE_SIZE`.